### ISSUE TITLE
[bug] cpython/3.10.0 unable to install due to conflict in libuuid

### ISSUE THREAD

**Issue by MaxKarel** [conan-io/conan#15182]
### Environment details

* Operating System+version: Ubuntu 20
* Compiler+version: gcc 9
* Conan version: 1.59.0
* Python version: Python 3.8.10

### Steps to reproduce

conan install cpython/3.10.0@

### Logs

```
Configuration:
[settings]
arch=x86_64
arch_build=x86_64
build_type=Debug
compiler=gcc
compiler.libcxx=libstdc++11
compiler.version=7.5
cuda=v11.7
os=Linux
os.distro=Ubuntu-20
os_build=Linux
[options]
[build_requires]
[env]

WARN: tcl/8.6.10: requirement zlib/[>=1.2.11 <2] overridden by tk/8.6.10 to zlib/1.2.13 
WARN: fontconfig/2.13.93: requirement expat/2.5.0 overridden by tk/8.6.10 to expat/2.4.1 
WARN: freetype/2.13.2: requirement zlib/[>=1.2.10 <2] overridden by fontconfig/2.13.93 to zlib/1.2.13 
WARN: libpng/1.6.40: requirement zlib/[>=1.2.11 <2] overridden by freetype/2.13.2 to zlib/1.2.13 
ERROR: At least two recipes provides the same functionality:
 - 'libuuid' provided by 'libuuid/1.0.3', 'util-linux-libuuid/2.39.2'

```

**Comment by RubenRBS** [comment]
Hi @MaxKarel, thanks for taking the time to report the issue

You're seeing a provides conflict because Conan Center Index moved from using `libuuid` to `util-linux-libuuid`, but the switchover has not yet been completed. In fact, CPython is the only remaining recipe with the old requirement.

I know that our contributors @Ahajha (and @valgur afaik!) is hard at work with the v2 port of that recipe - I've asked them to split the dependency update into a separate PR to fix this issue asap, so with a bit of luck, we'll be able to unblock you later today :)

**Comment by Ahajha** [comment]
I just opened up the PR: https://github.com/conan-io/conan-center-index/pull/21426

**Comment by samuel-emrys** [comment]
Also relevant: https://github.com/conan-io/conan-center-index/pull/18558
